src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -357,13 +357,21 @@ def _flexible_server_params(command_group):
             help="The availability zone information of the standby server when high availability is enabled."
         )
 
-        high_availability_arg_type = CLIArgumentType(
+        pg_high_availability_arg_type = CLIArgumentType(
             arg_type=get_enum_type(['Enabled', 'Disabled']),
             options_list=['--high-availability'],
             help='Enable or disable high availability feature. '
                  'Default value is Disabled. High availability can only be set during flexible server create time'
         )
 
+        mysql_high_availability_arg_type = CLIArgumentType(
+            arg_type=get_enum_type(['ZoneRedundant', 'SameZone', 'Disabled']),
+            options_list=['--high-availability'],
+            help='Enable (ZoneRedundant or SameZone) or disable high availability feature. '
+                 'Default value is Disabled. High availability can only be set during flexible server create time. '
+                 'Allowed values: ZoneRedundant, SameZone, Disabled'
+        )
```

Comment:
Please add this breaking change to release note.
Since your PR has made several changes, you could edit the **History Notes** part in PR description to list them all.